### PR TITLE
Added validation for table data in import-file-test and fixed the requirement of file-opts flags

### PR DESCRIPTION
**Pull request by sanyamsinghal**

### CODE REVIEW

File: migtests/tests/import-file/run-import-file-test [hunk 1]
```diff
@@ -55,17 +55,17 @@ main() {
 	# Test for fileOpts Flags having quote_char as single quote
 	step "Import data file: t1_quote_char.csv -> t1_quote_char"
 	import_data_file --data-dir ${TEST_DIR} --format csv --delimiter '|' \
-			--file-table-map "t1_quote_char.csv:t1_quote_char"
+			--file-table-map "t1_quote_char.csv:t1_quote_char" --file-opts "quote_char='"
```
Comment (amit-yb) [inline migtests/tests/import-file/run-import-file-test:58]: Can you enhance the `t1_quote_char.csv` file to have a string containing single quote. I expect the default escape character (`"`) to be used to specify a single quote inside a string.

File: migtests/tests/import-file/validate [hunk 2]
```diff
@@ -17,10 +17,17 @@ EXPECTED = {
  	"t1_quote_escape_char2": 3,
 }
 
+T1_QUOTE_CHAR = set([(1,None,None,'Hello World',), (2,None,None,'Hello World',), (3,None,None,'Hello World',)])
+T1_QUOTE_ESCAPE_CHAR = set([(1,None,None,"'Hello World'"), (2,None,None,"'Hello World'"), (3,None,None,"'Hello World'")])
```
Comment (amit-yb) [inline migtests/tests/import-file/validate:21]: Thanks for strengthening the check. 👍

**Review by amit-yb — APPROVED** [review]
Mostly looks good. Please, consider adding one more test enhancement as suggested in a comment.

File: migtests/tests/import-file/run-import-file-test (hunk 1 above)
Comment (sanyamsinghal) [reply migtests/tests/import-file/run-import-file-test:58]: Done

**Pull request merged**